plugins/polio/js/src/domains/Calendar/campaignCalendar/CampaignsFilters.tsx

```diff
@@ -312,17 +317,29 @@ export const CampaignsFilters: FunctionComponent<Props> = ({
                     </>
                 )}
                 {!disableOnlyDeleted && (
-                    <InputComponent
-                        keyValue="showOnlyDeleted"
-                        onChange={(key, value) => {
-                            setShowOnlyDeleted(value);
-                        }}
-                        value={showOnlyDeleted}
-                        type="checkbox"
-                        label={MESSAGES.showOnlyDeleted}
-                    />
+                    <>
+                        <InputComponent
+                            keyValue="showOnlyDeleted"
+                            onChange={(key, value) => {
+                                setShowOnlyDeleted(value);
+                            }}
+                            value={showOnlyDeleted}
+                            type="checkbox"
+                            label={MESSAGES.showOnlyDeleted}
+                        />
+                        <InputComponent
```

Comment:
Does this checkbox have to be inside the `disableOnlyDeleted` condition?

Reply:
Yes this checkbox should be inside disableOnlyDeleted because disableOnlyDeleted avoid to display the checkbox on calendar filter